company-proximity-navigator/package-lock.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Can you remove all the package-lock.json files in the project and add them to a `.gitignore` please?